requirements/quality-extra.txt

```diff
@@ -0,0 +1,2 @@
+pylint
+edx-lint                  # edX pylint rules and plugins
```

Comment:
What is the purpose for the quality-extra.txt file?
These are both also in quality.txt.

Same question about dev-extra.txt above.